clients/da-vinci-client/src/main/java/com/linkedin/davinci/blob/BlobSnapshotManager.java

```diff
@@ -0,0 +1,139 @@
+package com.linkedin.davinci.blob;
```

Comment:
Can you please also move this to a new package `blobtransfer`? I think it's consistent to call it `blobtransfer` everywhere.